### ISSUE TITLE
side bar doesn't render html

### ISSUE THREAD

**Issue by pabrams** [lakshaykaushik/PublishHTMLReport#10]
When I click Dashboard, the html never renders.  It renders fine when clicking Publish HTML Report (which auto selects Dashboard).  The other options don't render either.

**Comment by lakshaykaushik** [comment]
> When I click Dashboard, the html never renders. It renders fine when clicking Publish HTML Report (which auto selects Dashboard). The other options don't render much either.

It should work. It works fine in my azdo org.
Can you share me some steps so that I can reproduce what you're observing?

**Comment by pabrams** [comment]
Sure... so here's some version info:
- Azure Devops 2019 
- I was using Edge earlier, now Chrome

1. Run build
2. Click HTML tab, see nice tables and a pie chart
3. Click left nav "Dashboard":
![image](https://user-images.githubusercontent.com/1376688/100490202-a2181d00-30e7-11eb-8abb-1598a28dfca1.png)
4. Click around some more in the left nav (nothing rendering):
![image](https://user-images.githubusercontent.com/1376688/100490222-c83dbd00-30e7-11eb-8ca3-2daa5dc302a4.png)
5. Click Test Tab, then HTML tab  again - looks fine (as in first screenshot)

**Comment by pabrams** [comment]
Or is this because I have no data for those pages?

**Comment by pabrams** [comment]
If I get a chance, I'll try debugging this myself...

**Comment by lakshaykaushik** [comment]
1. How are you generating the jmeter reports folder?
2. Are you able to download the reports folder and open index.html in a browser.
3. Can you please list the folder structure inside the reports folder. Linux command: tree .

**Comment by pabrams** [comment]
1. `jmeter -n -t JMETER-POC\JMETER_Sample.jmx -l results-$(Build.BuildId)\JMETER_Sample_test_results.jtl -e -o results-$(Build.BuildId)\`
2. 
Yes, everything looks great when I do that. 

3. 

```
 Directory of E:\Agent\15\_work\416\s\results-139143

2020-12-01  05:07 PM    <DIR>          .
2020-12-01  05:07 PM    <DIR>          ..
2020-12-01  05:07 PM    <DIR>          content
2020-12-01  05:07 PM             9,687 index.html
2020-12-01  05:07 PM            75,801 JMETER_Sample_test_results.jtl
2020-12-01  05:07 PM    <DIR>          sbadmin2-1.0.7
2020-12-01  05:07 PM            17,959 statistics.json
               3 File(s)        103,447 bytes

 Directory of E:\Agent\15\_work\416\s\results-139143\content

2020-12-01  05:07 PM    <DIR>          .
2020-12-01  05:07 PM    <DIR>          ..
2020-12-01  05:07 PM    <DIR>          css
2020-12-01  05:07 PM    <DIR>          js
2020-12-01  05:07 PM    <DIR>          pages
               0 File(s)              0 bytes

 Directory of E:\Agent\15\_work\416\s\results-139143\content\css

2020-12-01  05:07 PM    <DIR>          .
2020-12-01  05:07 PM    <DIR>          ..
2020-11-17  04:14 PM             1,152 dashboard.css
2020-11-17  04:14 PM            31,589 jquery-ui.min.css
2020-11-17  04:14 PM            15,552 jquery-ui.structure.min.css
2020-11-17  04:14 PM            14,440 jquery-ui.theme.min.css
2020-11-17  04:14 PM             1,419 legends.css
2020-11-17  04:14 PM             7,555 theme.blue.css
               6 File(s)         71,707 bytes

 Directory of E:\Agent\15\_work\416\s\results-139143\content\js

2020-12-01  05:07 PM    <DIR>          .
2020-12-01  05:07 PM    <DIR>          ..
2020-11-17  04:14 PM            16,821 curvedLines.js
2020-12-01  05:07 PM             1,747 customGraph.js
2020-12-01  05:07 PM            14,506 dashboard-commons.js
2020-12-01  05:07 PM            19,811 dashboard.js
2020-12-01  05:07 PM           648,741 graph.js
2020-11-17  04:14 PM            14,107 hashtable.js
2020-11-17  04:14 PM           253,680 jquery-ui.min.js
2020-11-17  04:14 PM             3,238 jquery.cookie.js
2020-11-17  04:14 PM             3,141 jquery.flot.stack.js
2020-11-17  04:14 PM             4,675 jquery.numberformatter-1.2.3.min.js
2020-11-17  04:14 PM            37,977 jquery.tablesorter.min.js
              11 File(s)      1,018,444 bytes

 Directory of E:\Agent\15\_work\416\s\results-139143\content\pages

2020-12-01  05:07 PM    <DIR>          .
2020-12-01  05:07 PM    <DIR>          ..
2020-12-01  05:07 PM             7,839 CustomsGraphs.html
2020-11-17  04:14 PM             4,944 icon-apache.png
2020-12-01  05:07 PM            32,020 OverTime.html
2020-12-01  05:07 PM            22,534 ResponseTimes.html
2020-12-01  05:07 PM            29,021 Throughput.html
               5 File(s)         96,358 bytes

 Directory of E:\Agent\15\_work\416\s\results-139143\sbadmin2-1.0.7

2020-12-01  05:07 PM    <DIR>          .
2020-12-01  05:07 PM    <DIR>          ..
2020-11-17  04:14 PM               895 bower.json
2020-12-01  05:07 PM    <DIR>          bower_components
2020-12-01  05:07 PM    <DIR>          dist
2020-12-01  05:07 PM    <DIR>          less
2020-11-17  04:14 PM             1,531 README.md
               2 File(s)          2,426 bytes

 Directory of E:\Agent\15\_work\416\s\results-139143\sbadmin2-1.0.7\bower_components

2020-12-01  05:07 PM    <DIR>          .
2020-12-01  05:07 PM    <DIR>          ..
2020-12-01  05:07 PM    <DIR>          bootstrap
2020-12-01  05:07 PM    <DIR>          flot
2020-12-01  05:07 PM    <DIR>          flot-axislabels
2020-12-01  05:07 PM    <DIR>          flot.tooltip
2020-12-01  05:07 PM    <DIR>          font-awesome
2020-12-01  05:07 PM    <DIR>          jquery
2020-12-01  05:07 PM    <DIR>          metisMenu
               0 File(s)              0 bytes

 Directory of E:\Agent\15\_work\416\s\results-139143\sbadmin2-1.0.7\bower_components\bootstrap

2020-12-01  05:07 PM    <DIR>          .
2020-12-01  05:07 PM    <DIR>          ..
2020-11-17  04:14 PM             1,206 .bower.json
2020-12-01  05:07 PM    <DIR>          dist
2020-11-17  04:14 PM             6,871 README.md
               2 File(s)          8,077 bytes

 Directory of E:\Agent\15\_work\416\s\results-139143\sbadmin2-1.0.7\bower_components\bootstrap\dist

2020-12-01  05:07 PM    <DIR>          .
2020-12-01  05:07 PM    <DIR>          ..
2020-12-01  05:07 PM    <DIR>          css
2020-12-01  05:07 PM    <DIR>          fonts
2020-12-01  05:07 PM    <DIR>          js
               0 File(s)              0 bytes

 Directory of E:\Agent\15\_work\416\s\results-139143\sbadmin2-1.0.7\bower_components\bootstrap\dist\css

2020-12-01  05:07 PM    <DIR>          .
2020-12-01  05:07 PM    <DIR>          ..
2020-11-17  04:14 PM           121,205 bootstrap.min.css
               1 File(s)        121,205 bytes

 Directory of E:\Agent\15\_work\416\s\results-139143\sbadmin2-1.0.7\bower_components\bootstrap\dist\fonts

2020-12-01  05:07 PM    <DIR>          .
2020-12-01  05:07 PM    <DIR>          ..
2020-11-17  04:14 PM            20,127 glyphicons-halflings-regular.eot
2020-11-17  04:14 PM           109,025 glyphicons-halflings-regular.svg
2020-11-17  04:14 PM            45,404 glyphicons-halflings-regular.ttf
2020-11-17  04:14 PM            23,424 glyphicons-halflings-regular.woff
2020-11-17  04:14 PM            18,028 glyphicons-halflings-regular.woff2
               5 File(s)        216,008 bytes

 Directory of E:\Agent\15\_work\416\s\results-139143\sbadmin2-1.0.7\bower_components\bootstrap\dist\js

2020-12-01  05:07 PM    <DIR>          .
2020-12-01  05:07 PM    <DIR>          ..
2020-11-17  04:14 PM            37,051 bootstrap.min.js
               1 File(s)         37,051 bytes

 Directory of E:\Agent\15\_work\416\s\results-139143\sbadmin2-1.0.7\bower_components\flot

2020-12-01  05:07 PM    <DIR>          .
2020-12-01  05:07 PM    <DIR>          ..
2020-11-17  04:14 PM               450 .bower.json
2020-11-17  04:14 PM                38 .travis.yml
2020-11-17  04:14 PM            55,070 API.md
2020-11-17  04:14 PM               122 component.json
2020-11-17  04:14 PM             3,340 CONTRIBUTING.md
2020-11-17  04:14 PM            43,371 excanvas.js
2020-11-17  04:14 PM            19,314 excanvas.min.js
2020-11-17  04:14 PM             3,312 FAQ.md
2020-11-17  04:14 PM               864 flot.jquery.json
2020-11-17  04:14 PM             6,331 jquery.colorhelpers.js
2020-11-17  04:14 PM             9,944 jquery.flot.canvas.js
2020-11-17  04:14 PM             6,223 jquery.flot.categories.js
2020-11-17  04:14 PM             5,595 jquery.flot.crosshair.js
2020-11-17  04:14 PM            12,967 jquery.flot.errorbars.js
2020-11-17  04:14 PM             5,483 jquery.flot.fillbetween.js
2020-11-17  04:14 PM             7,601 jquery.flot.image.js
2020-11-17  04:14 PM           126,139 jquery.flot.js
2020-11-17  04:14 PM            14,562 jquery.flot.navigate.js
2020-11-17  04:14 PM            24,629 jquery.flot.pie.js
2020-11-17  04:14 PM             3,373 jquery.flot.resize.js
2020-11-17  04:14 PM            13,501 jquery.flot.selection.js
2020-11-17  04:14 PM             7,278 jquery.flot.stack.js
2020-11-17  04:14 PM             2,576 jquery.flot.symbol.js
2020-11-17  04:14 PM             4,622 jquery.flot.threshold.js
2020-11-17  04:14 PM            12,200 jquery.flot.time.js
2020-11-17  04:14 PM               297 Makefile
2020-11-17  04:14 PM            41,316 NEWS.md
2020-11-17  04:14 PM               165 package.json
2020-11-17  04:14 PM             4,483 PLUGINS.md
2020-11-17  04:14 PM             3,892 README.md
              30 File(s)        439,058 bytes

 Directory of E:\Agent\15\_work\416\s\results-139143\sbadmin2-1.0.7\bower_components\flot-axislabels

2020-12-01  05:07 PM    <DIR>          .
2020-12-01  05:07 PM    <DIR>          ..
2020-11-17  04:14 PM               401 .bower.json
2020-11-17  04:14 PM            20,008 jquery.flot.axislabels.js
2020-11-17  04:14 PM             3,079 README.md
               3 File(s)         23,488 bytes

 Directory of E:\Agent\15\_work\416\s\results-139143\sbadmin2-1.0.7\bower_components\flot.tooltip

2020-12-01  05:07 PM    <DIR>          .
2020-12-01  05:07 PM    <DIR>          ..
2020-11-17  04:14 PM               574 .bower.json
2020-11-17  04:14 PM               224 bower.json
2020-12-01  05:07 PM    <DIR>          js
               2 File(s)            798 bytes

 Directory of E:\Agent\15\_work\416\s\results-139143\sbadmin2-1.0.7\bower_components\flot.tooltip\js

2020-12-01  05:07 PM    <DIR>          .
2020-12-01  05:07 PM    <DIR>          ..
2020-11-17  04:14 PM            19,415 excanvas.min.js
2020-11-17  04:14 PM           126,139 jquery.flot.js
2020-11-17  04:14 PM            19,748 jquery.flot.tooltip.js
2020-11-17  04:14 PM             7,391 jquery.flot.tooltip.min.js
2020-11-17  04:14 PM            19,437 jquery.flot.tooltip.source.js
               5 File(s)        192,130 bytes

 Directory of E:\Agent\15\_work\416\s\results-139143\sbadmin2-1.0.7\bower_components\font-awesome

2020-12-01  05:07 PM    <DIR>          .
2020-12-01  05:07 PM    <DIR>          ..
2020-11-17  04:14 PM               732 .bower.json
2020-11-17  04:14 PM               469 .npmignore
2020-11-17  04:14 PM               435 bower.json
2020-12-01  05:07 PM    <DIR>          css
2020-12-01  05:07 PM    <DIR>          fonts
2020-12-01  05:07 PM    <DIR>          less
2020-12-01  05:07 PM    <DIR>          scss
               3 File(s)          1,636 bytes

 Directory of E:\Agent\15\_work\416\s\results-139143\sbadmin2-1.0.7\bower_components\font-awesome\css

2020-12-01  05:07 PM    <DIR>          .
2020-12-01  05:07 PM    <DIR>          ..
2020-11-17  04:14 PM            28,323 font-awesome.css
2020-11-17  04:14 PM            21,987 font-awesome.min.css
               2 File(s)         50,310 bytes

 Directory of E:\Agent\15\_work\416\s\results-139143\sbadmin2-1.0.7\bower_components\font-awesome\fonts

2020-12-01  05:07 PM    <DIR>          .
2020-12-01  05:07 PM    <DIR>          ..
2020-11-17  04:14 PM            56,006 fontawesome-webfont.eot
2020-11-17  04:14 PM           287,526 fontawesome-webfont.svg
2020-11-17  04:14 PM           112,160 fontawesome-webfont.ttf
2020-11-17  04:14 PM            65,452 fontawesome-webfont.woff
2020-11-17  04:14 PM            85,908 FontAwesome.otf
               5 File(s)        607,052 bytes

 Directory of E:\Agent\15\_work\416\s\results-139143\sbadmin2-1.0.7\bower_components\font-awesome\less

2020-12-01  05:07 PM    <DIR>          .
2020-12-01  05:07 PM    <DIR>          ..
2020-11-17  04:14 PM               346 bordered-pulled.less
2020-11-17  04:14 PM               429 core.less
2020-11-17  04:14 PM                42 extras.less
2020-11-17  04:14 PM               125 fixed-width.less
2020-11-17  04:14 PM               482 font-awesome.less
2020-11-17  04:14 PM            35,007 icons.less
2020-11-17  04:14 PM               383 larger.less
2020-11-17  04:14 PM               396 list.less
2020-11-17  04:14 PM               917 mixins.less
2020-11-17  04:14 PM               698 path.less
2020-11-17  04:14 PM               642 rotated-flipped.less
2020-11-17  04:14 PM               611 spinning.less
2020-11-17  04:14 PM               496 stacked.less
2020-11-17  04:14 PM            16,090 variables.less
              14 File(s)         56,664 bytes

 Directory of E:\Agent\15\_work\416\s\results-139143\sbadmin2-1.0.7\bower_components\font-awesome\scss

2020-12-01  05:07 PM    <DIR>          .
2020-12-01  05:07 PM    <DIR>          ..
2020-11-17  04:14 PM               422 font-awesome.scss
2020-11-17  04:14 PM               348 _bordered-pulled.scss
2020-11-17  04:14 PM               430 _core.scss
2020-11-17  04:14 PM             1,282 _extras.scss
2020-11-17  04:14 PM               126 _fixed-width.scss
2020-11-17  04:14 PM            35,556 _icons.scss
2020-11-17  04:14 PM               388 _larger.scss
2020-11-17  04:14 PM               397 _list.scss
2020-11-17  04:14 PM               931 _mixins.scss
2020-11-17  04:14 PM               709 _path.scss
2020-11-17  04:14 PM               692 _rotated-flipped.scss
2020-11-17  04:14 PM               612 _spinning.scss
2020-11-17  04:14 PM               502 _stacked.scss
2020-11-17  04:14 PM            16,153 _variables.scss
              14 File(s)         58,548 bytes

 Directory of E:\Agent\15\_work\416\s\results-139143\sbadmin2-1.0.7\bower_components\jquery

2020-12-01  05:07 PM    <DIR>          .
2020-12-01  05:07 PM    <DIR>          ..
2020-11-17  04:14 PM               774 .bower.json
2020-11-17  04:14 PM               462 bower.json
2020-12-01  05:07 PM    <DIR>          dist
               2 File(s)          1,236 bytes

 Directory of E:\Agent\15\_work\416\s\results-139143\sbadmin2-1.0.7\bower_components\jquery\dist

2020-12-01  05:07 PM    <DIR>          .
2020-12-01  05:07 PM    <DIR>          ..
2020-11-17  04:14 PM           282,115 jquery.js
2020-11-17  04:14 PM            86,929 jquery.min.js
2020-11-17  04:14 PM           132,382 jquery.min.map
               3 File(s)        501,426 bytes

 Directory of E:\Agent\15\_work\416\s\results-139143\sbadmin2-1.0.7\bower_components\metisMenu

2020-12-01  05:07 PM    <DIR>          .
2020-12-01  05:07 PM    <DIR>          ..
2020-11-17  04:14 PM               938 .bower.json
2020-12-01  05:07 PM    <DIR>          dist
2020-11-17  04:14 PM             2,593 README.md
               2 File(s)          3,531 bytes

 Directory of E:\Agent\15\_work\416\s\results-139143\sbadmin2-1.0.7\bower_components\metisMenu\dist

2020-12-01  05:07 PM    <DIR>          .
2020-12-01  05:07 PM    <DIR>          ..
2020-11-17  04:14 PM               790 metisMenu.min.css
2020-11-17  04:14 PM             1,887 metisMenu.min.js
               2 File(s)          2,677 bytes

 Directory of E:\Agent\15\_work\416\s\results-139143\sbadmin2-1.0.7\dist

2020-12-01  05:07 PM    <DIR>          .
2020-12-01  05:07 PM    <DIR>          ..
2020-12-01  05:07 PM    <DIR>          css
2020-12-01  05:07 PM    <DIR>          js
               0 File(s)              0 bytes

 Directory of E:\Agent\15\_work\416\s\results-139143\sbadmin2-1.0.7\dist\css

2020-12-01  05:07 PM    <DIR>          .
2020-12-01  05:07 PM    <DIR>          ..
2020-11-17  04:14 PM             7,453 sb-admin-2.css
               1 File(s)          7,453 bytes

 Directory of E:\Agent\15\_work\416\s\results-139143\sbadmin2-1.0.7\dist\js

2020-12-01  05:07 PM    <DIR>          .
2020-12-01  05:07 PM    <DIR>          ..
2020-11-17  04:14 PM             1,217 sb-admin-2.js
               1 File(s)          1,217 bytes

 Directory of E:\Agent\15\_work\416\s\results-139143\sbadmin2-1.0.7\less

2020-12-01  05:07 PM    <DIR>          .
2020-12-01  05:07 PM    <DIR>          ..
2020-11-17  04:14 PM                11 mixins.less
2020-11-17  04:14 PM             6,324 sb-admin-2.less
2020-11-17  04:14 PM               356 variables.less
               3 File(s)          6,691 bytes

     Total Files Listed:
             128 File(s)      3,628,638 bytes
              86 Dir(s)  213,494,427,648 bytes free
##[section]Finishing: Command Line Script
```

4.  

Task 1:
```
steps:
- script: 'E:\Appstage\apache-jmeter-5.2.1\bin\jmeter -n -t JMETER-POC\JMETER_Sample.jmx -l results-$(Build.BuildId)\JMETER_Sample_test_results.jtl -e -o results-$(Build.BuildId)\'
  failOnStderr: true
  displayName: 'Run jmeter_sample.jmx'
```

Task 2:
```
steps:
- task: LakshayKaushik.PublishHTMLReports.publishhtmlreport.publishhtmlreport@1
  displayName: publishhtmlreport
  inputs:
    htmlType: Jmeter
    JmeterReportsPath: '$(Build.SourcesDirectory)/results-$(Build.BuildId)'
```

**Comment by pabrams** [comment]
Here's some more console information that looks suspect; some 404s in there. This is on F5
```
[Intervention] Slow network is detected. See https://www.chromestatus.com/feature/5636954674692096 for more details. Fallback font will be used while loading: http://ado.intra.dmz/_static/tfs/Dev17.M153.5/_ext/ms.vss-features/ui-core-content/Components/Icon/fonts/AzDevMDL2.woff
ms.vss-build-web.buildresult.es6.qTJGl3o20d3GLbqS.min.js:1 Hub data: {build: {…}, definition: {…}, timeline: {…}, rawData: {…}, isLoading: false, …}
ms.vss-features.signalr.es6.W2c2cBpCyEsNSeJU.min.js:1 SignalR: moved from disconnected to connecting for /ProjectCollection/_apis/22092882-d4d8-46f0-bd8d-95e18864b74a
ms.vss-build-web.buildresult.es6.qTJGl3o20d3GLbqS.min.js:1 Additional Hub data: {changes: Array(1), repositoryName: "TS-Perf-PoC"}
index.html:1 [Intervention] Slow network is detected. See https://www.chromestatus.com/feature/5636954674692096 for more details. Fallback font will be used while loading: http://ado.intra.dmz/_apis/public/gallery/publisher/LakshayKaushik/extension/PublishHTMLReports/1.1.43/assetbyname/sbadmin2-1.0.7/bower_components/font-awesome/fonts/fontawesome-webfont.woff?v=4.2.0
DevTools failed to load SourceMap: Could not load content for http://ado.intra.dmz/_apis/public/gallery/publisher/LakshayKaushik/extension/PublishHTMLReports/1.1.43/assetbyname/sbadmin2-1.0.7/bower_components/bootstrap/dist/css/bootstrap.min.css.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
index.html:1 GET http://ado.intra.dmz/_apis/public/gallery/publisher/LakshayKaushik/extension/PublishHTMLReports/1.1.43/assetbyname/sbadmin2-1.0.7/bower_components/font-awesome/fonts/fontawesome-webfont.woff?v=4.2.0 net::ERR_ABORTED 404 (Not Found)
index.html:1 [Intervention] Slow network is detected. See https://www.chromestatus.com/feature/5636954674692096 for more details. Fallback font will be used while loading: http://ado.intra.dmz/_apis/public/gallery/publisher/LakshayKaushik/extension/PublishHTMLReports/1.1.43/assetbyname/sbadmin2-1.0.7/bower_components/font-awesome/fonts/fontawesome-webfont.ttf?v=4.2.0
index.html:1 GET http://ado.intra.dmz/_apis/public/gallery/publisher/LakshayKaushik/extension/PublishHTMLReports/1.1.43/assetbyname/sbadmin2-1.0.7/bower_components/font-awesome/fonts/fontawesome-webfont.ttf?v=4.2.0 net::ERR_ABORTED 404 (Not Found)
ms.vss-features.signalr.es6.W2c2cBpCyEsNSeJU.min.js:1 SignalR: moved from connecting to connected for /ProjectCollection/_apis/22092882-d4d8-46f0-bd8d-95e18864b74a
ms.vss-features.signalr.es6.W2c2cBpCyEsNSeJU.min.js:1 SignalR: connected!
Content?bundle=vss-bundle-async-vDyjzMXFZ5UcoLaGueucVi5g-aLujWc6Z_crjo3pYqNY=:1950 Please enable FF Webaccess.DistributedTask.KSB to register knockout secure binding.
Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248 TypeError: Cannot read property 'appendChild' of null
    at tab.js:78
    at c (Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:233)
    at Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:233
    at t.u.promiseDispatch (Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:233)
    at Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:233
    at MessagePort.t (Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:233)
(anonymous) @ Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248
t @ Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248
setTimeout (async)
t @ Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248
n.attachQPromiseErrorHandler @ Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248
n.initialize @ Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248
(anonymous) @ Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248
l @ jquery.min.js:2
c @ jquery.min.js:2
setTimeout (async)
(anonymous) @ jquery.min.js:2
u @ jquery.min.js:2
add @ jquery.min.js:2
(anonymous) @ jquery.min.js:2
Deferred @ jquery.min.js:2
then @ jquery.min.js:2
w.fn.ready @ jquery.min.js:2
w.fn.init @ jquery.min.js:2
w @ jquery.min.js:2
(anonymous) @ Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248
execCb @ require.min.js:6
check @ require.min.js:6
enable @ require.min.js:6
enable @ require.min.js:6
(anonymous) @ require.min.js:6
(anonymous) @ require.min.js:6
u @ require.min.js:6
enable @ require.min.js:6
enable @ require.min.js:6
(anonymous) @ require.min.js:6
(anonymous) @ require.min.js:6
u @ require.min.js:6
enable @ require.min.js:6
init @ require.min.js:6
fi @ require.min.js:6
completeLoad @ require.min.js:6
onScriptLoad @ require.min.js:6
load (async)
t.load @ require.min.js:6
load @ require.min.js:6
load @ require.min.js:6
fetch @ require.min.js:6
check @ require.min.js:6
enable @ require.min.js:6
enable @ require.min.js:6
(anonymous) @ require.min.js:6
(anonymous) @ require.min.js:6
u @ require.min.js:6
enable @ require.min.js:6
init @ require.min.js:6
fi @ require.min.js:6
completeLoad @ require.min.js:6
onScriptLoad @ require.min.js:6
load (async)
t.load @ require.min.js:6
load @ require.min.js:6
load @ require.min.js:6
fetch @ require.min.js:6
check @ require.min.js:6
enable @ require.min.js:6
enable @ require.min.js:6
(anonymous) @ require.min.js:6
(anonymous) @ require.min.js:6
u @ require.min.js:6
enable @ require.min.js:6
init @ require.min.js:6
(anonymous) @ require.min.js:6
setTimeout (async)
t.nextTick @ require.min.js:6
f @ require.min.js:6
requirejs @ require.min.js:6
(anonymous) @ index.html:47
(anonymous) @ VSS.SDK.min.js:3
w @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
y @ VSS.SDK.min.js:3
y @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
load (async)
y @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
load (async)
y @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
load (async)
y @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
load (async)
y @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
load (async)
y @ VSS.SDK.min.js:3
ht @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
n._wrapCallback @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
setTimeout (async)
n._resolve @ VSS.SDK.min.js:3
resolve @ VSS.SDK.min.js:3
t.onMessage @ VSS.SDK.min.js:3
n._handleMessageReceived @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
postMessage (async)
_sendRpcMessage @ ms.vss-web.ext-content.es6.U__nyOY219fITqYye.min.js:1
_success @ ms.vss-web.ext-content.es6.U__nyOY219fITqYye.min.js:1
(anonymous) @ ms.vss-web.ext-content.es6.U__nyOY219fITqYye.min.js:1
Promise.then (async)
invokeMethod @ ms.vss-web.ext-content.es6.U__nyOY219fITqYye.min.js:1
invokeMethodByName @ ms.vss-web.ext-content.es6.U__nyOY219fITqYye.min.js:1
onMessage @ ms.vss-web.ext-content.es6.U__nyOY219fITqYye.min.js:1
_handleMessageReceived @ ms.vss-web.ext-content.es6.U__nyOY219fITqYye.min.js:1
postMessage (async)
t._sendRpcMessage @ VSS.SDK.min.js:3
t.invokeRemoteMethod @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
setTimeout (async)
nt @ VSS.SDK.min.js:3
(anonymous) @ index.html:39
Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248 TypeError: Cannot read property 'appendChild' of null
    at tab.js:98
    at c (Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:233)
    at Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:233
    at t.u.promiseDispatch (Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:233)
    at Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:233
    at MessagePort.t (Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:233)
(anonymous) @ Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248
t @ Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248
setTimeout (async)
t @ Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248
n.attachQPromiseErrorHandler @ Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248
n.initialize @ Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248
(anonymous) @ Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248
l @ jquery.min.js:2
c @ jquery.min.js:2
setTimeout (async)
(anonymous) @ jquery.min.js:2
u @ jquery.min.js:2
add @ jquery.min.js:2
(anonymous) @ jquery.min.js:2
Deferred @ jquery.min.js:2
then @ jquery.min.js:2
w.fn.ready @ jquery.min.js:2
w.fn.init @ jquery.min.js:2
w @ jquery.min.js:2
(anonymous) @ Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248
execCb @ require.min.js:6
check @ require.min.js:6
enable @ require.min.js:6
enable @ require.min.js:6
(anonymous) @ require.min.js:6
(anonymous) @ require.min.js:6
u @ require.min.js:6
enable @ require.min.js:6
enable @ require.min.js:6
(anonymous) @ require.min.js:6
(anonymous) @ require.min.js:6
u @ require.min.js:6
enable @ require.min.js:6
init @ require.min.js:6
fi @ require.min.js:6
completeLoad @ require.min.js:6
onScriptLoad @ require.min.js:6
load (async)
t.load @ require.min.js:6
load @ require.min.js:6
load @ require.min.js:6
fetch @ require.min.js:6
check @ require.min.js:6
enable @ require.min.js:6
enable @ require.min.js:6
(anonymous) @ require.min.js:6
(anonymous) @ require.min.js:6
u @ require.min.js:6
enable @ require.min.js:6
init @ require.min.js:6
fi @ require.min.js:6
completeLoad @ require.min.js:6
onScriptLoad @ require.min.js:6
load (async)
t.load @ require.min.js:6
load @ require.min.js:6
load @ require.min.js:6
fetch @ require.min.js:6
check @ require.min.js:6
enable @ require.min.js:6
enable @ require.min.js:6
(anonymous) @ require.min.js:6
(anonymous) @ require.min.js:6
u @ require.min.js:6
enable @ require.min.js:6
init @ require.min.js:6
(anonymous) @ require.min.js:6
setTimeout (async)
t.nextTick @ require.min.js:6
f @ require.min.js:6
requirejs @ require.min.js:6
(anonymous) @ index.html:47
(anonymous) @ VSS.SDK.min.js:3
w @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
y @ VSS.SDK.min.js:3
y @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
load (async)
y @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
load (async)
y @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
load (async)
y @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
load (async)
y @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
load (async)
y @ VSS.SDK.min.js:3
ht @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
n._wrapCallback @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
setTimeout (async)
n._resolve @ VSS.SDK.min.js:3
resolve @ VSS.SDK.min.js:3
t.onMessage @ VSS.SDK.min.js:3
n._handleMessageReceived @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
postMessage (async)
_sendRpcMessage @ ms.vss-web.ext-content.es6.U__nyOY219fITqYye.min.js:1
_success @ ms.vss-web.ext-content.es6.U__nyOY219fITqYye.min.js:1
(anonymous) @ ms.vss-web.ext-content.es6.U__nyOY219fITqYye.min.js:1
Promise.then (async)
invokeMethod @ ms.vss-web.ext-content.es6.U__nyOY219fITqYye.min.js:1
invokeMethodByName @ ms.vss-web.ext-content.es6.U__nyOY219fITqYye.min.js:1
onMessage @ ms.vss-web.ext-content.es6.U__nyOY219fITqYye.min.js:1
_handleMessageReceived @ ms.vss-web.ext-content.es6.U__nyOY219fITqYye.min.js:1
postMessage (async)
t._sendRpcMessage @ VSS.SDK.min.js:3
t.invokeRemoteMethod @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
setTimeout (async)
nt @ VSS.SDK.min.js:3
(anonymous) @ index.html:39
Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248 TypeError: Cannot read property 'appendChild' of null
    at tab.js:138
    at c (Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:233)
    at Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:233
    at t.u.promiseDispatch (Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:233)
    at Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:233
    at MessagePort.t (Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:233)
(anonymous) @ Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248
t @ Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248
setTimeout (async)
t @ Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248
n.attachQPromiseErrorHandler @ Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248
n.initialize @ Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248
(anonymous) @ Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248
l @ jquery.min.js:2
c @ jquery.min.js:2
setTimeout (async)
(anonymous) @ jquery.min.js:2
u @ jquery.min.js:2
add @ jquery.min.js:2
(anonymous) @ jquery.min.js:2
Deferred @ jquery.min.js:2
then @ jquery.min.js:2
w.fn.ready @ jquery.min.js:2
w.fn.init @ jquery.min.js:2
w @ jquery.min.js:2
(anonymous) @ Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248
execCb @ require.min.js:6
check @ require.min.js:6
enable @ require.min.js:6
enable @ require.min.js:6
(anonymous) @ require.min.js:6
(anonymous) @ require.min.js:6
u @ require.min.js:6
enable @ require.min.js:6
enable @ require.min.js:6
(anonymous) @ require.min.js:6
(anonymous) @ require.min.js:6
u @ require.min.js:6
enable @ require.min.js:6
init @ require.min.js:6
fi @ require.min.js:6
completeLoad @ require.min.js:6
onScriptLoad @ require.min.js:6
load (async)
t.load @ require.min.js:6
load @ require.min.js:6
load @ require.min.js:6
fetch @ require.min.js:6
check @ require.min.js:6
enable @ require.min.js:6
enable @ require.min.js:6
(anonymous) @ require.min.js:6
(anonymous) @ require.min.js:6
u @ require.min.js:6
enable @ require.min.js:6
init @ require.min.js:6
fi @ require.min.js:6
completeLoad @ require.min.js:6
onScriptLoad @ require.min.js:6
load (async)
t.load @ require.min.js:6
load @ require.min.js:6
load @ require.min.js:6
fetch @ require.min.js:6
check @ require.min.js:6
enable @ require.min.js:6
enable @ require.min.js:6
(anonymous) @ require.min.js:6
(anonymous) @ require.min.js:6
u @ require.min.js:6
enable @ require.min.js:6
init @ require.min.js:6
(anonymous) @ require.min.js:6
setTimeout (async)
t.nextTick @ require.min.js:6
f @ require.min.js:6
requirejs @ require.min.js:6
(anonymous) @ index.html:47
(anonymous) @ VSS.SDK.min.js:3
w @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
y @ VSS.SDK.min.js:3
y @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
load (async)
y @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
load (async)
y @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
load (async)
y @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
load (async)
y @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
load (async)
y @ VSS.SDK.min.js:3
ht @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
n._wrapCallback @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
setTimeout (async)
n._resolve @ VSS.SDK.min.js:3
resolve @ VSS.SDK.min.js:3
t.onMessage @ VSS.SDK.min.js:3
n._handleMessageReceived @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
postMessage (async)
_sendRpcMessage @ ms.vss-web.ext-content.es6.U__nyOY219fITqYye.min.js:1
_success @ ms.vss-web.ext-content.es6.U__nyOY219fITqYye.min.js:1
(anonymous) @ ms.vss-web.ext-content.es6.U__nyOY219fITqYye.min.js:1
Promise.then (async)
invokeMethod @ ms.vss-web.ext-content.es6.U__nyOY219fITqYye.min.js:1
invokeMethodByName @ ms.vss-web.ext-content.es6.U__nyOY219fITqYye.min.js:1
onMessage @ ms.vss-web.ext-content.es6.U__nyOY219fITqYye.min.js:1
_handleMessageReceived @ ms.vss-web.ext-content.es6.U__nyOY219fITqYye.min.js:1
postMessage (async)
t._sendRpcMessage @ VSS.SDK.min.js:3
t.invokeRemoteMethod @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
setTimeout (async)
nt @ VSS.SDK.min.js:3
(anonymous) @ index.html:39
Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248 TypeError: Cannot read property 'appendChild' of null
    at tab.js:118
    at c (Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:233)
    at Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:233
    at t.u.promiseDispatch (Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:233)
    at Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:233
    at MessagePort.t (Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:233)
(anonymous) @ Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248
t @ Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248
setTimeout (async)
t @ Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248
n.attachQPromiseErrorHandler @ Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248
n.initialize @ Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248
(anonymous) @ Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248
l @ jquery.min.js:2
c @ jquery.min.js:2
setTimeout (async)
(anonymous) @ jquery.min.js:2
u @ jquery.min.js:2
add @ jquery.min.js:2
(anonymous) @ jquery.min.js:2
Deferred @ jquery.min.js:2
then @ jquery.min.js:2
w.fn.ready @ jquery.min.js:2
w.fn.init @ jquery.min.js:2
w @ jquery.min.js:2
(anonymous) @ Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248
execCb @ require.min.js:6
check @ require.min.js:6
enable @ require.min.js:6
enable @ require.min.js:6
(anonymous) @ require.min.js:6
(anonymous) @ require.min.js:6
u @ require.min.js:6
enable @ require.min.js:6
enable @ require.min.js:6
(anonymous) @ require.min.js:6
(anonymous) @ require.min.js:6
u @ require.min.js:6
enable @ require.min.js:6
init @ require.min.js:6
fi @ require.min.js:6
completeLoad @ require.min.js:6
onScriptLoad @ require.min.js:6
load (async)
t.load @ require.min.js:6
load @ require.min.js:6
load @ require.min.js:6
fetch @ require.min.js:6
check @ require.min.js:6
enable @ require.min.js:6
enable @ require.min.js:6
(anonymous) @ require.min.js:6
(anonymous) @ require.min.js:6
u @ require.min.js:6
enable @ require.min.js:6
init @ require.min.js:6
fi @ require.min.js:6
completeLoad @ require.min.js:6
onScriptLoad @ require.min.js:6
load (async)
t.load @ require.min.js:6
load @ require.min.js:6
load @ require.min.js:6
fetch @ require.min.js:6
check @ require.min.js:6
enable @ require.min.js:6
enable @ require.min.js:6
(anonymous) @ require.min.js:6
(anonymous) @ require.min.js:6
u @ require.min.js:6
enable @ require.min.js:6
init @ require.min.js:6
(anonymous) @ require.min.js:6
setTimeout (async)
t.nextTick @ require.min.js:6
f @ require.min.js:6
requirejs @ require.min.js:6
(anonymous) @ index.html:47
(anonymous) @ VSS.SDK.min.js:3
w @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
y @ VSS.SDK.min.js:3
y @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
load (async)
y @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
load (async)
y @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
load (async)
y @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
load (async)
y @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
load (async)
y @ VSS.SDK.min.js:3
ht @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
n._wrapCallback @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
setTimeout (async)
n._resolve @ VSS.SDK.min.js:3
resolve @ VSS.SDK.min.js:3
t.onMessage @ VSS.SDK.min.js:3
n._handleMessageReceived @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
postMessage (async)
_sendRpcMessage @ ms.vss-web.ext-content.es6.U__nyOY219fITqYye.min.js:1
_success @ ms.vss-web.ext-content.es6.U__nyOY219fITqYye.min.js:1
(anonymous) @ ms.vss-web.ext-content.es6.U__nyOY219fITqYye.min.js:1
Promise.then (async)
invokeMethod @ ms.vss-web.ext-content.es6.U__nyOY219fITqYye.min.js:1
invokeMethodByName @ ms.vss-web.ext-content.es6.U__nyOY219fITqYye.min.js:1
onMessage @ ms.vss-web.ext-content.es6.U__nyOY219fITqYye.min.js:1
_handleMessageReceived @ ms.vss-web.ext-content.es6.U__nyOY219fITqYye.min.js:1
postMessage (async)
t._sendRpcMessage @ VSS.SDK.min.js:3
t.invokeRemoteMethod @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
setTimeout (async)
nt @ VSS.SDK.min.js:3
(anonymous) @ index.html:39
Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248 TypeError: Cannot read property 'appendChild' of null
    at tab.js:230
    at c (Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:233)
    at Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:233
    at t.u.promiseDispatch (Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:233)
    at Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:233
    at MessagePort.t (Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:233)
(anonymous) @ Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248
t @ Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248
setTimeout (async)
t @ Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248
n.attachQPromiseErrorHandler @ Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248
n.initialize @ Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248
(anonymous) @ Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248
l @ jquery.min.js:2
c @ jquery.min.js:2
setTimeout (async)
(anonymous) @ jquery.min.js:2
u @ jquery.min.js:2
add @ jquery.min.js:2
(anonymous) @ jquery.min.js:2
Deferred @ jquery.min.js:2
then @ jquery.min.js:2
w.fn.ready @ jquery.min.js:2
w.fn.init @ jquery.min.js:2
w @ jquery.min.js:2
(anonymous) @ Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248
execCb @ require.min.js:6
check @ require.min.js:6
enable @ require.min.js:6
enable @ require.min.js:6
(anonymous) @ require.min.js:6
(anonymous) @ require.min.js:6
u @ require.min.js:6
enable @ require.min.js:6
enable @ require.min.js:6
(anonymous) @ require.min.js:6
(anonymous) @ require.min.js:6
u @ require.min.js:6
enable @ require.min.js:6
init @ require.min.js:6
fi @ require.min.js:6
completeLoad @ require.min.js:6
onScriptLoad @ require.min.js:6
load (async)
t.load @ require.min.js:6
load @ require.min.js:6
load @ require.min.js:6
fetch @ require.min.js:6
check @ require.min.js:6
enable @ require.min.js:6
enable @ require.min.js:6
(anonymous) @ require.min.js:6
(anonymous) @ require.min.js:6
u @ require.min.js:6
enable @ require.min.js:6
init @ require.min.js:6
fi @ require.min.js:6
completeLoad @ require.min.js:6
onScriptLoad @ require.min.js:6
load (async)
t.load @ require.min.js:6
load @ require.min.js:6
load @ require.min.js:6
fetch @ require.min.js:6
check @ require.min.js:6
enable @ require.min.js:6
enable @ require.min.js:6
(anonymous) @ require.min.js:6
(anonymous) @ require.min.js:6
u @ require.min.js:6
enable @ require.min.js:6
init @ require.min.js:6
(anonymous) @ require.min.js:6
setTimeout (async)
t.nextTick @ require.min.js:6
f @ require.min.js:6
requirejs @ require.min.js:6
(anonymous) @ index.html:47
(anonymous) @ VSS.SDK.min.js:3
w @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
y @ VSS.SDK.min.js:3
y @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
load (async)
y @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
load (async)
y @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
load (async)
y @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
load (async)
y @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
load (async)
y @ VSS.SDK.min.js:3
ht @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
n._wrapCallback @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
setTimeout (async)
n._resolve @ VSS.SDK.min.js:3
resolve @ VSS.SDK.min.js:3
t.onMessage @ VSS.SDK.min.js:3
n._handleMessageReceived @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
postMessage (async)
_sendRpcMessage @ ms.vss-web.ext-content.es6.U__nyOY219fITqYye.min.js:1
_success @ ms.vss-web.ext-content.es6.U__nyOY219fITqYye.min.js:1
(anonymous) @ ms.vss-web.ext-content.es6.U__nyOY219fITqYye.min.js:1
Promise.then (async)
invokeMethod @ ms.vss-web.ext-content.es6.U__nyOY219fITqYye.min.js:1
invokeMethodByName @ ms.vss-web.ext-content.es6.U__nyOY219fITqYye.min.js:1
onMessage @ ms.vss-web.ext-content.es6.U__nyOY219fITqYye.min.js:1
_handleMessageReceived @ ms.vss-web.ext-content.es6.U__nyOY219fITqYye.min.js:1
postMessage (async)
t._sendRpcMessage @ VSS.SDK.min.js:3
t.invokeRemoteMethod @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
setTimeout (async)
nt @ VSS.SDK.min.js:3
(anonymous) @ index.html:39
Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248 TypeError: Cannot read property 'appendChild' of null
    at tab.js:188
    at c (Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:233)
    at Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:233
    at t.u.promiseDispatch (Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:233)
    at Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:233
    at MessagePort.t (Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:233)
(anonymous) @ Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248
t @ Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248
setTimeout (async)
t @ Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248
n.attachQPromiseErrorHandler @ Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248
n.initialize @ Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248
(anonymous) @ Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248
l @ jquery.min.js:2
c @ jquery.min.js:2
setTimeout (async)
(anonymous) @ jquery.min.js:2
u @ jquery.min.js:2
add @ jquery.min.js:2
(anonymous) @ jquery.min.js:2
Deferred @ jquery.min.js:2
then @ jquery.min.js:2
w.fn.ready @ jquery.min.js:2
w.fn.init @ jquery.min.js:2
w @ jquery.min.js:2
(anonymous) @ Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248
execCb @ require.min.js:6
check @ require.min.js:6
enable @ require.min.js:6
enable @ require.min.js:6
(anonymous) @ require.min.js:6
(anonymous) @ require.min.js:6
u @ require.min.js:6
enable @ require.min.js:6
enable @ require.min.js:6
(anonymous) @ require.min.js:6
(anonymous) @ require.min.js:6
u @ require.min.js:6
enable @ require.min.js:6
init @ require.min.js:6
fi @ require.min.js:6
completeLoad @ require.min.js:6
onScriptLoad @ require.min.js:6
load (async)
t.load @ require.min.js:6
load @ require.min.js:6
load @ require.min.js:6
fetch @ require.min.js:6
check @ require.min.js:6
enable @ require.min.js:6
enable @ require.min.js:6
(anonymous) @ require.min.js:6
(anonymous) @ require.min.js:6
u @ require.min.js:6
enable @ require.min.js:6
init @ require.min.js:6
fi @ require.min.js:6
completeLoad @ require.min.js:6
onScriptLoad @ require.min.js:6
load (async)
t.load @ require.min.js:6
load @ require.min.js:6
load @ require.min.js:6
fetch @ require.min.js:6
check @ require.min.js:6
enable @ require.min.js:6
enable @ require.min.js:6
(anonymous) @ require.min.js:6
(anonymous) @ require.min.js:6
u @ require.min.js:6
enable @ require.min.js:6
init @ require.min.js:6
(anonymous) @ require.min.js:6
setTimeout (async)
t.nextTick @ require.min.js:6
f @ require.min.js:6
requirejs @ require.min.js:6
(anonymous) @ index.html:47
(anonymous) @ VSS.SDK.min.js:3
w @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
y @ VSS.SDK.min.js:3
y @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
load (async)
y @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
load (async)
y @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
load (async)
y @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
load (async)
y @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
load (async)
y @ VSS.SDK.min.js:3
ht @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
n._wrapCallback @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
setTimeout (async)
n._resolve @ VSS.SDK.min.js:3
resolve @ VSS.SDK.min.js:3
t.onMessage @ VSS.SDK.min.js:3
n._handleMessageReceived @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
postMessage (async)
_sendRpcMessage @ ms.vss-web.ext-content.es6.U__nyOY219fITqYye.min.js:1
_success @ ms.vss-web.ext-content.es6.U__nyOY219fITqYye.min.js:1
(anonymous) @ ms.vss-web.ext-content.es6.U__nyOY219fITqYye.min.js:1
Promise.then (async)
invokeMethod @ ms.vss-web.ext-content.es6.U__nyOY219fITqYye.min.js:1
invokeMethodByName @ ms.vss-web.ext-content.es6.U__nyOY219fITqYye.min.js:1
onMessage @ ms.vss-web.ext-content.es6.U__nyOY219fITqYye.min.js:1
_handleMessageReceived @ ms.vss-web.ext-content.es6.U__nyOY219fITqYye.min.js:1
postMessage (async)
t._sendRpcMessage @ VSS.SDK.min.js:3
t.invokeRemoteMethod @ VSS.SDK.min.js:3
(anonymous) @ VSS.SDK.min.js:3
setTimeout (async)
nt @ VSS.SDK.min.js:3
(anonymous) @ index.html:39
Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:248 TypeError: Cannot read property 'appendChild' of null
    at tab.js:163
    at c (Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:233)
    at Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:233
    at t.u.promiseDispatch (Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:233)
    at Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:233
    at MessagePort.t (Content?bundle=vss-bundle-ext-core-v_S4Z47nwbh6DZ3lCIOrXDL4WbkhPFU9KExlwHhyV57k=:233)

**Comment by lakshaykaushik** [comment]
I will test this with your jmeter version and see if i can reproduce. 
Meanwhile, what is the extension version you are using?

**Comment by lakshaykaushik** [comment]
If its possible, would be great if you can zip and send results-$(Build.BuildId) folder to me. Make sure you dont include any customer or any important information.

**Comment by lakshaykaushik** [comment]
I tested with 5.2 version of Jmeter and it works just fine. 
So, would be nice to have your reports folder and the extension version you are using because i cannot still reproduce it.

The 404 you are seeing are due to some incompatible files are removed.

**Comment by pabrams** [comment]
[drop.zip](https://github.com/lakshaykaushik/PublishHTMLReport/files/5629833/drop.zip)
Extension version is 1.1.43:
![image](https://user-images.githubusercontent.com/1376688/100885034-2ff55e80-3480-11eb-8c5b-c077a7404655.png)

**Comment by lakshaykaushik** [comment]
I have done some updates since 1.1.43 for race around conditions. can you try the latest version 
of the extension and confirm if you still see the same behaviour.

**Comment by pabrams** [comment]
The marketplace is still reporting 1.1.43 as the current version.  I don't think it would let us upgrade?  Iirc, there's a check that makes sure the version number is higher?  (I can't try it myself, I have to open a ticket for the ADO admins to do it)

**Comment by pabrams** [comment]
It does look like I'm missing some commits, and I can see version 1.1.44 in the git history, but as far as I can tell you haven't published it?

**Comment by lakshaykaushik** [comment]
Latest version is 52.

https://marketplace.visualstudio.com/items?itemName=LakshayKaushik.PublishHTMLReports. Checkout the more info section, maybe you need to uninstall and re-install if its stuck for you.

**Comment by pabrams** [comment]
Upgraded the extension, but still having the same issue.  When I click on one of the side menu items, the html doesn't render.  Here are some console messages I get on that event.  Also, including screenshot at the end.  Tried this in Edge and Chrome.

> 
> Refused to apply style from 'http://adoservername/_apis/public/gallery/publisher/LakshayKaushik/extension/PublishHTMLReports/1.1.52/assetbyname/css/legends.css' because its MIME type ('application/json') is not a supported stylesheet MIME type, and strict MIME checking is enabled.
> /_apis/public/gallery/publisher/LakshayKaushik/extension/PublishHTMLReports/1.1.52/assetbyname/ResponseTimes.html:1 Refused to apply style from 'http://adoservername/_apis/public/gallery/publisher/LakshayKaushik/extension/PublishHTMLReports/1.1.52/assetbyname/css/jquery-ui.structure.min.css' because its MIME type ('application/json') is not a supported stylesheet MIME type, and strict MIME checking is enabled.
> /_apis/public/gallery/publisher/LakshayKaushik/extension/PublishHTMLReports/1.1.52/assetbyname/ResponseTimes.html:1 Refused to apply style from 'http://adoservername/_apis/public/gallery/publisher/LakshayKaushik/extension/PublishHTMLReports/1.1.52/assetbyname/css/jquery-ui.theme.min.css' because its MIME type ('application/json') is not a supported stylesheet MIME type, and strict MIME checking is enabled.
> /_apis/public/gallery/publisher/LakshayKaushik/extension/PublishHTMLReports/1.1.52/assetbyname/ResponseTimes.html:1 Refused to apply style from 'http://adoservername/_apis/public/gallery/publisher/LakshayKaushik/extension/PublishHTMLReports/1.1.52/assetbyname/css/jquery-ui.min.css' because its MIME type ('application/json') is not a supported stylesheet MIME type, and strict MIME checking is enabled.
> DevTools failed to load SourceMap: Could not load content for http://adoservername/_apis/public/gallery/publisher/LakshayKaushik/extension/PublishHTMLReports/1.1.52/assetbyname/sbadmin2-1.0.7/bower_components/bootstrap/dist/css/bootstrap.min.css.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
> font-awesome.min.css:1 GET http://adoservername/_apis/public/gallery/publisher/LakshayKaushik/extension/PublishHTMLReports/1.1.52/assetbyname/sbadmin2-1.0.7/bower_components/font-awesome/fonts/fontawesome-webfont.woff?v=4.2.0 net::ERR_ABORTED 404 (Not Found)
> 2bootstrap.min.css:1 GET http://adoservername/_apis/public/gallery/publisher/LakshayKaushik/extension/PublishHTMLReports/1.1.52/assetbyname/sbadmin2-1.0.7/bower_components/bootstrap/dist/fonts/glyphicons-halflings-regular.woff2 net::ERR_ABORTED 404 (Not Found)
> font-awesome.min.css:1 GET http://adoservername/_apis/public/gallery/publisher/LakshayKaushik/extension/PublishHTMLReports/1.1.52/assetbyname/sbadmin2-1.0.7/bower_components/font-awesome/fonts/fontawesome-webfont.ttf?v=4.2.0 net::ERR_ABORTED 404 (Not Found)
> bootstrap.min.css:1 GET http://adoservername/_apis/public/gallery/publisher/LakshayKaushik/extension/PublishHTMLReports/1.1.52/assetbyname/sbadmin2-1.0.7/bower_components/bootstrap/dist/fonts/glyphicons-halflings-regular.ttf net::ERR_ABORTED 404 (Not Found)

![image](https://user-images.githubusercontent.com/1376688/101075851-9063c900-3570-11eb-81f0-415b74485dc6.png)

**Comment by pabrams** [comment]
I had a glance at the source code, and I see some setTimeout ( , 1000) in there around each of those appendChild complaints in the console.  I'm guessing they are the culprits.

**Comment by pabrams** [comment]
This also seems suspicious; the string is "firstscriptname", but then later it's just "firstscript", which isn't found.
```
                    taskClient.getPlanAttachments(vsoContext.project.id, "build", build.orchestrationPlan.planId, "firstscriptname").then((taskAttachments) => {
                        $.each(taskAttachments, (index, taskAttachment) => {
                            if (taskAttachment._links && taskAttachment._links.self && taskAttachment._links.self.href) {
                                var recId = taskAttachment.recordId;
                                var timelineId = taskAttachment.timelineId;
                                taskClient.getAttachmentContent(vsoContext.project.id, "build", build.orchestrationPlan.planId, timelineId, recId, "firstscriptname", taskAttachment.name).then((attachementContent) => {
                                    function arrayBufferToString(buffer) {
                                        var arr = new Uint8Array(buffer);
                                        var str = String.fromCharCode.apply(String, arr);
                                        return str;
                                    }
                                    var first = arrayBufferToString(attachementContent);
                                    document.body.style.overflow = "visible";
                                    var s = document.createElement("script");
                                    s.innerHTML = first;
                                    s.async = false;
                                    //s.type = "text/javascript";
                                    document.getElementById("firstscript").appendChild(s);
                                });
                            }
                        });
                    });
```

**Comment by lakshaykaushik** [comment]
That is fine. 
Timeout may be a problem if you're on a very very slow internet connection. Timeouts were added to get around some race conditions, I didn't want the script to run till the pages render. If the pages are rendering slow the script timeout will trigger the script which won't do anything. 
I am guessing this is what is happening in your case.  But if you refresh multiple times, this may start working sometimes if it's because of the network speed. Can you check, what happens when you do hard refresh multiple times? Does it work sometime?

Script names are fine as they should be.

Can you send me the screenshot by doing inspect element and opening the html body of #document of the page which is not rendering?

**Comment by pabrams** [comment]
The problem with multiple refresh is it resets to a state where the bug does not exist.  The bug is when you use the side menu. Should using the side menu cause a postback?  

![image](https://user-images.githubusercontent.com/1376688/101085129-feae8880-357c-11eb-9173-3c0da0dfe46f.png)

**Comment by pabrams** [comment]
![image](https://user-images.githubusercontent.com/1376688/101085242-2dc4fa00-357d-11eb-8045-81f7871e3950.png)

**Comment by pabrams** [comment]
![image](https://user-images.githubusercontent.com/1376688/101085427-6c5ab480-357d-11eb-818f-179dd7763d64.png)

**Comment by lakshaykaushik** [comment]
I just rendered your drop.zip which you attached in this, it worked perfectly fine and all graphs and charts are rendering. 

Come on this: https://gitter.im/PublishHTMLReports/community?utm_source=share-link&utm_medium=link&utm_campaign=share-link

I will help you out.

**Comment by pabrams** [comment]
Yes, drop.zip worked fine for me too.

**Comment by pabrams** [comment]
I can't get to gitter, and anyway I'm sure it's the slow network, which means it's the 1-second timeouts that are the problem, right?  Should you just wait for the promise to resolve, or something?

**Comment by pabrams** [comment]
The issue is that all the files are there (as in drop.zip) but the extension does not render them.  It works fine if you don't run it through the extension, of course.

**Comment by pabrams** [comment]
__

**Comment by pabrams** [comment]
Why does your code reference "firstchild", etc., if those strings don't exist?

**Comment by lakshaykaushik** [comment]
Looks like you are behind a vpn or a proxy. Message me on gitter without vpn or something, I will see what can be done in your case. 
1 second timeout is enough for reasonable speeds.

**Comment by pabrams** [comment]
I can't message you on gitter, it's blocked by government, and I don't have reasonable speeds; I'm limited by our network. 
Using timeouts like that is a terrible practice.  You should do it properly, and wait for the promise to resolve.

**Comment by pabrams** [comment]
I don't think you can help me anymore.  But I can help you.

**Comment by lakshaykaushik** [comment]
If there are no timeouts, the whole rendering gets extremely slow while waiting for everything synchronously before the all the scripts run. 
But I agree with you, it's an issue while on slower networks, it should render howsoever the network is slow.
You can expect it to be solved soon.

**Comment by lakshaykaushik** [comment]
Closing this as it seems to be an env issue and haven't heard about this issue from anyone else who is using this extension.
